backend/data/src/main/kotlin/io/tolgee/dtos/request/notification/NotificationFilters.kt

```diff
@@ -0,0 +1,5 @@
+package io.tolgee.dtos.request.notification
+
+class NotificationFilters(
+  val unseenOnly: Boolean = false,
```

Comment:
The convention is to name it `filterSeen`. When you make it nullable you can have this options:
- null: make this filter ignored
- true: filter seen
- false: filter unseen

see `io.tolgee.repository.UserAccountRepositoryKt#USER_FILTERS` for reference